iphone/Classes/TiUtils.h

```diff
@@ -662,11 +662,23 @@ typedef enum {
 + (BOOL)isRetinaiPhone6Plus;
 
 /**
- Whether or not the current device has a 5.8 inch retina display (iPhone X).
- @return _YES_ if the current device has a 5.8 inch retina display, _NO_ otherwise.
+ Whether or not the current device has a 5.8 inch super retina display (iPhone X).
+ @return _YES_ if the current device has a 5.8 inch super retina display, _NO_ otherwise.
  */
 + (BOOL)isRetinaiPhoneX;
 
+/**
+ Whether or not the current device has a 6.5 inch super retina display (iPhone X Max).
+ @return _YES_ if the current device has a 6.5 inch super retina display, _NO_ otherwise.
+ */
++ (BOOL)isRetinaiPhoneXSMax;
+
+/**
+ Whether or not the current device has a 6.1 inch retina display (iPhone X Max).
```

Comment:
`iPhone X Max` -> `iPhone XR`
May want to consider adding the actual resolution/density values here in the comments as well?